pkg/controller/gittrackobject/handler_test.go

```diff
@@ -240,6 +246,12 @@ var _ = Describe("Handler Suite", func() {
 				child = testutils.ExampleClusterRoleBinding.DeepCopy()
 				Expect(testutils.SetGitTrackObjectInterfaceSpec(gto, child)).To(Succeed())
 
+				r = r.withValues(
+					"Namespace", gto.GetNamespace(),
+					"ChildName", gto.GetSpec().Name,
+					"ChildKind", gto.GetSpec().Kind,
+				)
```

Comment:
These should probably be updated to match the ones above? i.e. `{namespace, child name, child kind}`